app/connectors/httpParsers/GetAnnualIncomeSourcePeriodHttpParser.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.httpParsers
+
+import models.giftAid.SubmittedGiftAidModel
+import models.{ErrorModel}
+import play.api.Logging
+import play.api.http.Status._
+import uk.gov.hmrc.http.{HttpReads, HttpResponse}
+import utils.PagerDutyHelper.PagerDutyKeys._
+import utils.PagerDutyHelper.pagerDutyLog
+
+object GetAnnualIncomeSourcePeriodHttpParser extends APIParser with Logging {
+    type GetAnnualIncomeSourcePeriod = Either[ErrorModel, SubmittedGiftAidModel]
```

Comment:
Can `SubmittedGiftAidModel` be renamed to `AnnualIncomeSourcePeriod`?

Reply:
This is the existing model, which we will be using for new API too